### ISSUE TITLE
Additional field/space for 2FA backup codes

### ISSUE THREAD

**Issue by gabriele-v** [Rookiestyle/KeePassOTP#68]
## Summary
Most of sites that provides 2FA authentication, provides also additional offline codes.
It will be helpful to have another space in KeePassOTP where storing them.

## Added value
I'm using KeePassOTP saving in a separate DB, so it doesn't make sense to save backup codes into main DB (IE notes field) because otherwise the separate DB it's useless. It would be helpful to save them somewhere within the separate DB.

## Example
![image](https://user-images.githubusercontent.com/7106290/136688543-d42c5416-5b38-4698-9e96-cdcb388d7974.png)
![image](https://user-images.githubusercontent.com/7106290/136688591-618e7091-5ada-43b9-af44-3ac2dc4d1c80.png)

**Comment by Rookiestyle** [comment]
Backup / recovery codes can be used to restore access if required.
I'll add an option in the setup form to enter these as well.
If you use a separate db, these codes will be stored in the separate db too.
If you don't use a separate db, these codes will be stored in a protected field in the respective entry.



I'm not aware of offline codes, can you provide an example?

**Comment by Rookiestyle** [comment]
I was more interested to know whether your "offline codes" are something different than backup/recovery codes.
I know understand that there is no difference, thanks for clarifying.

The setup window will be reworked, current idea is to have 3 tabs
1. OTP secret (seed), Drag&Drop target for QR code (left side), Scan buton to search for QR code on screen (right side)
2. Backup codes
3. Advanced settings

**Comment by gabriele-v** [comment]
Yep sorry, for offline I was meaning when the 2FA generator it's not available, sorry for the misunderstanding.

**Comment by gabriele-v** [comment]
Cool, thank you!
